.github/workflows/publish.yml

```diff
@@ -91,6 +91,8 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           NUGET_API_TOKEN: ${{ secrets.NUGET_API_TOKEN }}
+          GPG_SECRET_KEY: ${{ secrets.GPG_SECRET_KEY }}
+          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
```

Comment:
I believe the presenting case here is the Maven target (https://github.com/getsentry/craft/pull/346). Will we ever want GPG signing for other targets? Does it make sense to use a global GPG key for all targets as the var name here suggests? Do we want to scope this down to Maven or somesuch?

Reply:
I was thinking about this, but I don't see a reason not to use a generic key here. Bruno mentioned something about .net, there is signing for iOS as well, so we may need it in the future.

Reply:
.NET code signing won't be done with GPG (it's a code signing cert and the dotnet built-in tooling)

Reply:
Okay, sounds like we're on it.